cdap-watchdog/src/test/java/co/cask/cdap/metrics/store/timeseries/FactTableTest.java

```diff
@@ -123,14 +123,14 @@ public void testBasics() throws Exception {
     expected.clear();
 
     // delete the metrics data at (timestamp + 20) resolution
-    scan = new FactScan(ts + resolution * 2, ts + resolution * 3, null, tagValues);
+    scan = new FactScan(ts + resolution * 2, ts + resolution * 4, null, tagValues);
```

Comment:
oh

At first I made a change to delete whole rows, so needed to make it *2 since rolltime is 2. Noww I can change it back.

good catch!